content/3_Silver/Intro_Bitwise.mdx

```diff
@@ -407,6 +407,52 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+```java
+import java.io.*;
+import java.util.*;
+
+public class XorPyramid {
+	public static void main(String[] args) throws IOException {
+		BufferedReader f = new BufferedReader(new InputStreamReader(System.in));
+		PrintWriter out = new PrintWriter(System.out);
+		StringTokenizer st = new StringTokenizer(f.readLine());
+
+		int n = Integer.parseInt(st.nextToken());
+
+		// Precompute the powers of 2 in the factorization of i!
+		int[] pref = new int[n + 1];
+		for (int i = 2; i <= n; i++) {
+			int num = i;
+			while (num % 2 == 0) {
+				pref[i]++;
+				num /= 2;
+			}
+			pref[i] += pref[i - 1];
+		}
+
+		int ans = 0;
+		st = new StringTokenizer(f.readLine());
+		for (int i = 0; i < n; i++) {
+			int num = Integer.parseInt(st.nextToken());
+			// Determine if the i-th term in Pascal's Triangle
+			// is odd by determining the number of 2s in the factorization of n
+			// choose k.
```

Comment:
```
/*
 * multiline 
 * comments
 * like
 * this
 */
```

Reply:
Here is the AC screenshot:

<img width="761" alt="Screenshot 2024-08-04 at 4 26 34 PM" src="https://github.com/user-attachments/assets/61033f06-bf24-4182-960c-7a044ea94eeb">

Thank you!